tests/acceptance/settings/profile/names-test.ts

```diff
@@ -52,15 +52,21 @@ module('Acceptance | settings | profile | name', hooks => {
         assert.dom('[data-test-mla-citation]').containsText(mla);
         await percySnapshot(assert);
         await fillIn('[data-test-given-name-field] input', '');
-        assert.dom('[data-test-citation-container]').doesNotExist();
+        assert.dom('[data-test-citation-container]').exists();
```

Comment:
This is new behavior because the form no longer live-updates the model. You have to submit changes to see changes.